parallel_cluster/conf/config

```diff
@@ -23,12 +23,40 @@ scheduler             = slurm
 initial_queue_size    = 1
 custom_ami            = ami-002b1d0f440e6caea
 
+[cluster umccr_dev]
+base_os               = alinux2
+vpc_settings          = umccr_dev_network
+# Definitely experimental
+fsx_settings          = lustrefs
+s3_read_resource      = *
+cluster_type          = spot
+# Need something substantial to hold the slurm database
+master_instance_type  = t2.medium
+# 16 CPUs and 64 Gb
+compute_instance_type = m5.4xlarge
+ec2_iam_role          = parallelcluster-ec2-instance-role
+scheduler             = slurm
+initial_queue_size    = 1
+# Basic ami with installations of
+# R, python3.8, conda, pip, ruby, golang, rust
+# conda has been initialised to the ec2-user
+custom_ami            = ami-0b98c49201c9a1e75
+
 [vpc network]
 vpc_id = vpc-0020eaf6bc27c98a4
 master_subnet_id = subnet-07a7800b072d25eb3
 compute_subnet_id = subnet-0506a40a906ea0156
 use_public_ips = true
 
+[vpc umccr_dev_network]
+# Default vpc for dev account
+vpc_id = vpc-00eafc63c0dfca266
+# Compute subnet ids take the same subnet as the master
+# Our default public subnet
+master_subnet_id = subnet-0fab038b0341872f1
+# Elastic IP address is associated to the master instance.
+use_public_ips = true
```

Comment:
We might want to `false` this since users should login via SSM.

Reply:
Will test early this week to see if there are any unintended consequences.